provider/gce/environ_network.go

```diff
@@ -125,9 +125,8 @@ func (e *environ) getInstanceSubnets(
 	for _, iface := range ifaces {
 		if subnetIds.Include(string(iface.ProviderSubnetId)) {
 			results = append(results, makeSubnetInfo(
-				// TODO (manadart 2019-07-11): Move InterfaceInfo to core and remove casting.
-				corenetwork.Id(iface.ProviderSubnetId),
-				corenetwork.Id(iface.ProviderNetworkId),
+				iface.ProviderSubnetId,
```

Comment:
golinter said was unnecessary

Reply:
I obviously missed some of these that I added before deciding to move the `Id` type into `core/network`.
Thanks for the catch.